src/frontend/src/redux/records/actions.ts

```diff
@@ -0,0 +1,33 @@
+import { Dispatch } from 'redux';
+import apiService from '../../service/api-service';
+import { AxiosResponse } from 'axios';
+import { LOAD_SEARCH_RECORDS, LOAD_SEARCH_RECORDS_LOADING } from './types';
+
+export function loadSearchRecords(
+  first_name: string,
+  last_name: string,
+  birthday: string
+): any {
+  return (dispatch: Dispatch) => {
+    dispatch({
+      type: LOAD_SEARCH_RECORDS_LOADING
+    });
+    return apiService(dispatch, {
+      url: '/api/search',
+      data: {
+        first_name: first_name,
+        middle_name: '',
+        last_name: last_name,
+        birth_date: birthday
+      },
+      method: 'post',
+      authenticated: true,
+      withCredentials: true
+    }).then((response: AxiosResponse) => {
+      dispatch({
```

Comment:
Should probably have some kind of validation for the response format here,

Reply:
Were you thinking of checking the response against a JSON schema?

Reply:
@KentShikama nah, it can be something simple, just that from my experience it will save someone time in the future if we have some basic checks to make sure there won't be a property access error on `response.data.data.record` in case the backend doesn't send the correct format, or someone changes something in the future. If something goes wrong then the app ideally shouldn't silently fail or fail in an unexpected way. Just thinking 80/20 rule, doesn't have to be checked against a schema (though that's fine too).

Reply:
Ok I've just checked that `data` and `record` properties exist under `response.data` for now.